homeassistant/components/uptime/sensor.py

```diff
@@ -65,16 +53,4 @@ def state(self):
 
     async def async_update(self):
         """Update the state of the sensor."""
-        delta = dt_util.now() - self.initial
-        div_factor = 3600
-
-        if self.unit_of_measurement == "days":
-            div_factor *= 24
-        elif self.unit_of_measurement == "minutes":
-            div_factor /= 60
-        elif self.unit_of_measurement == "seconds":
-            div_factor /= 3600
-
-        delta = delta.total_seconds() / div_factor
-        self._state = round(delta, 2)
-        _LOGGER.debug("New value: %s", delta)
+        self._state = self.initial
```

Comment:
? This sensor doesn't update anymore now, right?

Reply:
Correct, just on boot. No other need for uptime to be recalculated